.github/workflows/deploy-cf-prune-latest.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy CF prune-latest
+on:
+  pull_request:
+    paths:
+      - 'cf/prune-latest'
+      - '.github/workflows/deploy-cf-prune-latest.yml'
+  push:
+    paths:
```

Comment:
Maybe also limit the branch?
```yaml
push:
  branches:
        - main
   path:
```

Reply:
Why would we limit it to `main`? We want the `--dry-check` to run on pushes to branches as well.